packages/oui-form-actions/src/form-actions.component.js

```diff
@@ -0,0 +1,15 @@
+import controller from "./form-actions.controller.js";
+import template from "./form-actions.html";
+
+export default {
+    template,
+    controller,
+    bindings: {
+        submitText: "@?",
+        cancelText: "@?",
+        onSubmit: "&",
+        onCancel: "&",
```

Comment:
Optional callback?

Reply:
I had thought about make a default callback which will do $windows.history.back. But that's not the proper things to do IMO.

Reply:
Without a "?" it's already optional. It's replaced with angular.noop